molecule/testinfra/common/test_automatic_updates.py

```diff
@@ -154,12 +154,18 @@ def test_unattended_upgrades_functional(host):
         f" origin=Ubuntu,archive={distro}, origin=Ubuntu,archive={distro}-security"
         f", origin=Ubuntu,archive={distro}-updates, origin=SecureDrop,codename={distro}"
     )
-    expected_result = (
-        "No packages found that can be upgraded unattended and no pending auto-removals"
-    )
 
+    all_good = "No packages found that can be upgraded unattended and no pending auto-removals"
     assert expected_origins in c.stdout
-    assert expected_result in c.stdout
+    if distro == "focal":
+        assert all_good in c.stdout
+    else:  # noqa: PLR5501
```

Comment:
Disabled because otherwise it rolls up the check into: `if distro == "focal" or all_good in c.stdout` which is technically equivalent but harder to read the meaning of.